pkg/memory/meminfo.go

```diff
@@ -52,9 +52,13 @@ func MemTotalNormal() (uint64, error) {
 	if time.Since(t) < 60*time.Second {
 		return total, nil
 	}
+	return totalMem()
+}
+
+func totalMem() (uint64, error) {
```

Comment:
BTW, why not just keep the same name as TiDB?

Reply:
It breaks the lint:  https://revive.run/r#confusing-naming  Method 'memTotalNormal' differs only by capitalization to function 'MemTotalNormal' in the same source file  
  /home/runner/work/pd/pd/pkg/memory/meminfo.go:58:6